lib/rspec/core/option_parser.rb

```diff
@@ -140,8 +140,18 @@ def parser(options)
           options[:full_backtrace] = true
         end
 
-        parser.on('-c', '--[no-]color', '--[no-]colour', 'Enable color in the output.') do |o|
-          options[:color] = o
+        parser.on('-c', '--color', '--colour', 'Deprecated: enable color in the output.') do |_o|
+          options[:color] = true
+        end
+
+        parser.on('--force-color', '--force-colour', 'Force the output to be in color.') do |_o|
+          options[:force_color] = true
+          options.delete(:no_color)
```

Comment:
https://github.com/rspec/rspec-core/pull/2321/files#diff-2bae3d9145ad46203f8a7fa5140ce8acL200 actually demonstrates that the last flag wins. The way this works is because the flags don't delete each other. Instead, the `parser` processes each flag in turn, which sets and unsets _something else_: an entry in the `options` hash. That's why unsetting an option doesn't delete a flag.

I can definitely look into the aborting option. Question for backward compatibility: the `--color` and `--no-color` flags previously had this last-wins behavior, so I think it would be good to leave that in. So I'm thinking just abort in case `--force-color` and `--no-color` are used together. Sound good?